chain/messagepool/provider.go

```diff
@@ -39,10 +42,19 @@ type Provider interface {
 	IsLite() bool
 }
 
+type nonceCacheKey struct {
```

Comment:
is this the right name? it _may_ reference a nonce lookup error but from from a use-perspective ("getActorLite") it's in the path of getting the actor where the nonce is just a secondary step, so maybe this would be better as `actorLookupKey`